Missionframework/presets/init_presets.sqf

```diff
@@ -127,7 +127,7 @@ all_hostile_classnames = (land_vehicles_classnames + opfor_air + opfor_choppers
 air_vehicles_classnames = [] + opfor_choppers;
 KP_liberation_friendly_air_classnames = [];
 {air_vehicles_classnames pushback (_x select 0); KP_liberation_friendly_air_classnames pushback (_x select 0);} foreach air_vehicles;
-KP_liberation_friendly_air_classnames = KP_liberation_friendly_air_classnames - uavs;
+{ if(_x call F_isClassUAV) then { KP_liberation_friendly_air_classnames = KP_liberation_friendly_air_classnames - [_x] };} foreach KP_liberation_friendly_air_classnames;
```

Comment:
Guess this would be faster if you use

    KP_liberation_friendly_air_classnames = KP_liberation_friendly_air_classnames select {!(_x call F_isClassUAV)};

Will do a small test how faster it is. Well, this won't impact the mission with one select (and there are many many other things to optimize), but just as a heads up for future things. Also in the rewrite there is more `select` than `forEach`, as it's more performance friendly.

Reply:
Ok, thanks for the information I will remember to use _select_ for array filtering in the future.